gpu-operator/life-cycle-policy.rst

```diff
@@ -89,7 +89,8 @@ Refer to :ref:`Upgrading the NVIDIA GPU Operator` for more information.
      - ${version}
 
    * - NVIDIA GPU Driver
-     - | `570.133.20 <https://docs.nvidia.com/datacenter/tesla/tesla-release-notes-570-133-20/index.html>`_ (recommended), 
+     - | `570.148.08 <https://docs.nvidia.com/datacenter/tesla/tesla-release-notes-570-148-08/index.html>`_ (recommended)
```

Comment:
This will also be the default driver once `25.3.1` is live

Reply:
yes, i have the update in the 25.3.1 docs PR as default and recommended. This will be visible for just the 25.3.0 version.